wa/framework/target/runtime_config.py

```diff
@@ -939,6 +955,12 @@ def check_target(self):
     def validate_parameters(self):
         pass
 
+    def handle_screen_unlock(self):
+        if 'unlock_screen' in self.config and self.config['unlock_screen']:
```

Comment:
Can simplify that statement:
```
        if self.config.get('unlock_screen'):
```

Reply:
Nope. It throws an exception if I don't first check if `unlock_screen` is in `self.config`.

Reply:
I think this would be fixed by using `None` as the disabled case as this is what the setter method is checking for before populating `self.config`.